index.html

```diff
@@ -21,180 +22,176 @@
+                            <img src="Microverse-Student-Project-1/social media icons.png">
+                        </div>
+                    </div> 
+                    <div class="button-container">
+                        <div class="about-button">
+                            Get my resume
+                        </div>
+                    </div>
                 </div>
-            </section>
-            
-            <section class="vert-tabs">
-                <ul class="tab-list">
-                <li class="tab-item">
-                    <div class="tab-title"><span class="tab-text">Languages</span> <i class="fa-solid fa-arrow-down"></i></div>
-
-                    <div class="tab-lang"><img src="Microverse-Student-Project-1/Ellipse 1.png"><span class="tab-text">JavaScript</span></div>
-                    <div class="tab-lang"><img src="Microverse-Student-Project-1/Ellipse 1(1).png"><span class="tab-text">HTML</span></div>
-                    <div class="tab-lang"><img src="Microverse-Student-Project-1/Ellipse 1(2).png"><span class="tab-text">CSS</span></div>
-                </li>
 
-                <li class="tab-item">
-                    <span class="tab-title">Frameworks <i class="fa-solid fa-arrow-right"></i></span>
-                    <div class="line"></div>
-                </li>
-                <li class="tab-item">
-                    <span class="tab-title">Skills <i class="fa-solid fa-arrow-right"></i></span>
-                    <div class="line"></div>
-                </li>
-                </ul>
-            </section>
+                <ul class="tab-list">
+                    <li class="tab-item">
+                        <div class="tab-title"><span class="tab-text">Languages</span> <i class="fa-solid fa-arrow-down"></i></div>
+                        <div class="tab-lang"><img src="Microverse-Student-Project-1/Ellipse 1.png"><span class="tab-text">JavaScript</span></div>
+                        <div class="tab-lang"><img src="Microverse-Student-Project-1/Ellipse 1(1).png"><span class="tab-text">HTML</span></div>
+                        <div class="tab-lang"><img src="Microverse-Student-Project-1/Ellipse 1(2).png"><span class="tab-text">CSS</span></div>
+                    </li>
+                    <li class="tab-item">
+                        <span class="tab-title">Frameworks <i class="fa-solid fa-arrow-right"></i></span>
+                        <div class="line"></div>
+                    </li>
+                    <li class="tab-item">
+                        <span class="tab-title">Skills <i class="fa-solid fa-arrow-right"></i></span>
+                        <div class="line"></div>
+                    </li>
+                    </ul>   
+            </section>     
+                
             <div class="space">
-
             </div>
-
+            <!-- CONTACT -->
             <footer class="contact">
                 <div class="contact-container">
```

Comment:
- [ ] Kindly invest some time here too, to fix some bugs that shields your design prowess. I can't find your button, the background images is not showing and I think you need to work on some elements e.g. `width`, `padding`, `margin`, etc. as revealed with the image below:

![image](https://user-images.githubusercontent.com/99715640/213843024-1909cf5a-031a-4da2-b0a6-8b837908ad67.png)